service/src/main/java/bio/terra/tanagra/app/controller/CohortsApiController.java

```diff
@@ -67,8 +67,8 @@ public ResponseEntity<ApiCohort> createCohort(String studyId, ApiCohortCreateInf
             Cohort.builder()
                 .displayName(body.getDisplayName())
                 .description(body.getDescription())
-                .underlay(body.getUnderlayName()),
-            SpringAuthentication.getCurrentUser().getEmail());
+                .underlay(body.getUnderlayName())
+                .createdBy(SpringAuthentication.getCurrentUser().getEmail()));
```

Comment:
Useremail is only used for this field in nested functions. Add it to builder and remove the param

Reply:
Correct, the activity log used cohort.getCreatedBy(), which might also be more descriptive imo